readthedocs/proxito/middleware.py

```diff
@@ -59,6 +62,21 @@ def map_host_to_project_slug(request):
             request, 'core/dns-404.html', context={'host': host}, status=400
         )
 
+    if external_domain in host:
+        # Serve custom versions on external-host-domain
+        if external_domain_parts == host_parts[1:]:
+            try:
+                project_slug, version_slug = host_parts[0].split('--', 1)
+                request.external_domain = True
+                request.host_version_slug = version_slug
+                log.debug('Proxito External Domain: host=%s', host)
```

Comment:
We don't call it `external versions` other places, I don't quite understand why we should here.

Reply:
Because they are external versions. We have `Version.type = 'external'` and and `Version.objects.external`, etc. So, basically all the versions that come from Pull Request we are naming them "External Versions". 

That's my point.